services/venice-server/src/main/java/com/linkedin/venice/grpc/VeniceGrpcServer.java

```diff
@@ -0,0 +1,46 @@
+package com.linkedin.venice.grpc;
+
+import com.linkedin.venice.exceptions.VeniceException;
+import io.grpc.Grpc;
+import io.grpc.Server;
+import io.grpc.ServerInterceptors;
+import java.io.IOException;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+
+public class VeniceGrpcServer {
+  private static final Logger LOGGER = LogManager.getLogger(VeniceGrpcServer.class);
+  private final Server server;
+  private final int port;
+
+  public VeniceGrpcServer(VeniceGrpcServerConfig config) {
+    port = config.getPort();
+    server = Grpc.newServerBuilderForPort(config.getPort(), config.getCredentials())
```

Comment:
This seems to be too simple..
I wonder whether there is a way to configure thread pool size or not.

Reply:
It is, however, it is not recommended based on the workload and may not be as performant. We can pass a custom executor when creating the gRPC server to best suit the workload.

Reply:
Can you leave a `TODO` here?